OpenSim/Simulation/Model/Force.h

```diff
@@ -84,7 +84,7 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Force, ModelComponent);
     * thread that is running in parallel with other forces
     * that marked this flag as true.
     */
```

Comment:
Can you improve the doxygen comment above so that it says `Simbody` instead of `SimBody`?

Can you also put the note here about not parallelizing forces that aren't threadsafe?

I'm also thinking it might be a good idea to put a note in the detailed description of this class that the calculations can be parallelized.